account_tax_analysis/account_tax_analysis_view.xml

```diff
@@ -26,18 +26,27 @@
           <newline/>
           <group expand="1" string="Group By...">
             <filter string="Partner" icon="terp-partner"
+            		name="group_by_partner"
                     domain="[]"
                     context="{'group_by': 'partner_id'}"/>
             <separator orientation="vertical"/>
             <filter string="Account"
+            		name="group_by_account"
                     icon="terp-folder-green"
                     context="{'group_by': 'account_id'}"/>
             <separator orientation="vertical"/>
             <filter string="Period"
+            	    name="group_by_period"
                     icon="terp-go-month"
                     domain="[]"
                     context="{'group_by': 'period_id'}"/>
+            <filter string="Account / Tax"
```

Comment:
Hello @pedrobaeza 
Thanks for the review,
I have put a special filter, to be able to define an order in the group by .

I can't find a exemple of favorite definition in the code, could you please point me an example ?

Thanks for your help,

Vincent